tests/Countries/IndiaTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Spatie\Holidays\Tests\Countries;
+
+use Carbon\CarbonImmutable;
+use Spatie\Holidays\Holidays;
+
+it('can calculate india holidays', function ($year) {
+    CarbonImmutable::setTestNow($year.'-01-01');
+    $holidays = Holidays::for(country: 'in')->get();
+    expect($holidays)
+        ->toBeArray()
+        ->not()->toBeEmpty();
+
+    expect(formatDates($holidays))->toMatchSnapshot();
+})->with([1970, 1971, 1972, 1973, 1974, 1975, 1976, 1977, 1978, 1979, 1980, 1981, 1982, 1983, 1984, 1985, 1986, 1987, 1988, 1989, 1990, 1991, 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029, 2030, 2031, 2032, 2033, 2034, 2035, 2036, 2037]);
```

Comment:
I don't think we need this many tests. Better to pick a few ones and verify if the generated snapshots are correct.

Reply:
Additionally, for specific behaviour you can also make a specific test and assert, instead of a full snapshot test

Reply:
@Nielsvanpach  Above changes are done.